### PR TITLE
Add more in-dialog validations.

### PR DESCRIPTION
**Pull request by ILMTitan**

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindowContent.xaml [hunk 1]
```diff
@@ -62,8 +62,14 @@
                        Style="{StaticResource CommonLabelStyle}"/>
                 <TextBox x:Name="_trafficAllocation"
                          Grid.Column="1"
-                         Text="{Binding Allocation, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
                          Style="{StaticResource CommonTextBoxStyle}">
+                    <TextBox.Text>
+                        <Binding
```
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindowContent.xaml]: Why not the inline version? Is more readable.

File: GoogleCloudExtension/GoogleCloudExtension/AddWindowsCredential/AddwindowsCredentialWindowContent.xaml [hunk 2]
```diff
@@ -36,10 +36,18 @@
...
-                     x:Name="_userName"
-                     Margin="0,0,0,24"
-                     Style="{StaticResource CommonTextBoxStyle}">
+            <TextBox
+                x:Name="_userName"
+                Margin="0,0,0,24"
+                Style="{StaticResource CommonTextBoxStyle}">
+                
+                <TextBox.Text>
+                    <Binding
```
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/AddWindowsCredential/AddwindowsCredentialWindowContent.xaml]: I'd rather have the inline version of the Binding, is more readable.

File: GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindowContent.xaml (hunk 1 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindowContent.xaml]: There's no need to modify the source code to meet an arbitrary line length. Specially in Xaml where the parsing is different depending on what you are doing, that is the binding expression is faster to parse than the element version.

I'd rather leave it as is, have the line long and move on.

Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/AddTrafficSplit/AddTrafficSplitWindowContent.xaml]: Done

File: GoogleCloudExtension/GoogleCloudExtension/AddWindowsCredential/AddwindowsCredentialWindowContent.xaml (hunk 2 above)
Comment (ILMTitan) [inline GoogleCloudExtension/GoogleCloudExtension/AddWindowsCredential/AddwindowsCredentialWindowContent.xaml]: Done

**Pull request merged**